integration/kubernetes/alluxio-master.yaml

```diff
@@ -0,0 +1,81 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: alluxio-master
+  labels:
+    app: alluxio
+spec:
+  ports:
+  - port: 19998
+    name: rpc
+  - port: 19999
+    name: web
+  clusterIP: None
+  selector:
+    app: alluxio-master
+---
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: alluxio-master
+spec:
+  selector:
+    matchLabels:
+      app: alluxio-master
+  serviceName: "alluxio-master"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: alluxio-master
+    spec:
+      hostNetwork: true
+      hostPID: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+        - name: alluxio-master
+          image: alluxio/alluxio:1.6.1
```

Comment:
does k8s pull from dockerhub by default?

Reply:
correct. the image can be overridden as <registry>/<repo>/<image>:<tag> if using some other registry